roles/core/time/molecule/default/prepare.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Prepare
+  hosts: all
+
+  tasks:
+    - name: "update cache apt"
+      apt: update_cache=yes
+      when: ansible_os_family == "Debian"
+
+    - name: "update cache yum"
```

Comment:
Not needed. We need to be as close as possible than production where we don't force a cache update.
We have to update the cache in the Ubuntu image because the maintainer of the image is running `rm -Rf /var/lib/apt/lists/*` in the Dockerfile.